src/widget/container.rs

```diff
@@ -64,4 +79,40 @@ where
             None => rectangle.into_styled(style).draw(display),
         }
     }
+
+    fn draw_children(
+        &self,
+        display: &mut Display,
+        origin: Point,
+        max_size: Size,
+    ) -> Result<Size, Display::Error> {
+        let mut total_size = Size::new(max_size.width, 0);
```

Comment:
I might name this `current_size`? I wasn't clear at first that this is sort of an "accumulator" as you're looping